lib/broccoli/ember-app.js

```diff
@@ -1176,8 +1191,19 @@ EmberApp.prototype.testFiles = function(coreTestTree) {
 
   var emberCLITree = this._processedEmberCLITree();
 
-  var testJs = this.concatFiles(mergeTrees([external, coreTestTree]), {
-    inputFiles: this.legacyTestFilesToAppend,
+  var coreExt = mergeTrees([external, coreTestTree])
+
+  coreExt = stewMap(coreExt, function(content, relativePath){
+    if (legacyTestFilesToAppend.indexOf(relativePath) > -1) {
```

Comment:
i haven't taken much time to think about this or this PR, but I'm inclined to think it is something that should happen during sourceMapConcat